cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -273,6 +317,19 @@ func (k *KubernetesClient) GetManifest(group, version, kind, name, namespace str
 	return string(data), nil
 }
 
+func (k *KubernetesClient) Restart(group, version, kind, name, namespace string) (any, error) {
```

Comment:
You can remove any from the return 
```suggestion
func (k *KubernetesClient) Restart(group, version, kind, name, namespace string) error {
```

Reply:
removed